src/rtl_433_ESP.cpp

```diff
@@ -32,7 +32,11 @@
 #if defined(RF_MODULE_SCK) && defined(RF_MODULE_MISO) && \
     defined(RF_MODULE_MOSI) && defined(RF_MODULE_CS)
 #  include <SPI.h>
+#  if CONFIG_IDF_TARGET_ESP32C3 || CONFIG_IDF_TARGET_ESP32S3
+SPIClass newSPI(FSPI);
+#  elif CONFIG_IDF_TARGET_ESP32
```

Comment:
Shouldn't this be an else rather than a else if to preserve original behaviour?

Reply:
Yes and no.
I use `elif` because the actual don't work with rpi2040, es8266 and libretiny. I'm working to extend the use of the library in esphome ecosystem.

If you prefer I will replace with else for now.

L.

Reply:
Leave it as else for now, and when the others are ready make the adjustment.